src/pallene/pallenelib.lua

```diff
@@ -48,6 +48,7 @@ return [==[
 
 /* Type tags */
 static const char *pallene_tag_name(int raw_tag);
+static const char *pallene_type_name(lua_State *L, const TValue *raw_type);
```

Comment:
The `TValue*` points to a Lua value, not to a raw type. Please name this parameter more similarly to how we name the other TValue parameters in the neighboring functions.